eng/SourceBuild.props

```diff
@@ -55,6 +55,7 @@
 
     <PropertyGroup>
       <InnerBuildArgs>$(InnerBuildArgs) /p:SourceBuildRuntimeIdentifier=$(TargetRuntimeIdentifier)</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) /p:CrossgenOutput=false</InnerBuildArgs>
```

Comment:
Could we use regular crossgen?

Reply:
What will set `$(SourceBuildUseMonoRuntime)`❔ I don't see it in Arcade, at least the commit I've got locally

Reply:
> Could we use regular crossgen?

I don't know. Maybe mono doesn't support R2R assemblies? I'm just guessing.
Disabling this is what we're doing on the mono specific architectures already:

https://github.com/dotnet/aspnetcore/blob/b9dae95a6c95d113c208ef83a33ca1a69912b00d/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj#L45-L47

> What will set $(SourceBuildUseMonoRuntime)grey_question I don't see it in Arcade, at least the commit I've got locally

It gets set by `installer`, see https://github.com/dotnet/installer/pull/14792.